src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -363,25 +364,12 @@ protected async Task<bool> SettingsFileHasChangedAsync()
         }
 
         /// <summary>
-        /// Helper function to set the new snapshot, update the active profile property to match what is in the snapshot,  and post
-        /// the changes to consumers.
+        /// Helper function to set the new snapshot and post the changes to consumers.
         /// </summary>
-        protected async Task FinishUpdateAsync(ILaunchSettings newSnapshot, bool ensureProfileProperty)
+        protected void FinishUpdateAsync(ILaunchSettings newSnapshot)
```

Comment:
>FinishUpdateAsync [](start = 23, length = 17)

Should the Async suffix be dropped from this method?

Reply:
Ah, yes.